src/apps/explorer/components/TransanctionBatchGraph/styled.ts

```diff
@@ -95,7 +95,12 @@ export function STYLESHEET(theme: DefaultTheme): Stylesheet[] {
         'background-color': theme.bg2,
       },
     },
-
+    {
+      selector: 'node[label].hover',
+      style: {
+        color: theme.orange,
```

Comment:
changing the cursor here you said doesn't work, right?

Reply:
Yep, it only accepts here their subset of CSS rules https://js.cytoscape.org/#style